pkg/chartrenderer/default.go

```diff
@@ -191,10 +191,10 @@ func (c *RenderedChart) AsSecretData() map[string][]byte {
 }
 
 // loadEmbeddedFS is a copy of chartutil.LoadDir with the difference that it uses an embed.FS.
```

Comment:
It is no longer `chartutil.LoadDir` in helm3.

```suggestion
// loadEmbeddedFS is a copy of helm.sh/helm/v3/pkg/chart/loader.LoadDir with the difference that it uses an embed.FS.
// Keep this func in sync with https://github.com/helm/helm/blob/v3.10.3/pkg/chart/loader/directory.go#L44-L120.
```

Can you also double-check whether this func is in sync with the func in helm3?